### DISCUSSION TITLE
如何完完全全删除/卸载该插件及其相关组件， 以便重装？

### DISCUSSION THREAD

**Discussion by aswirl4** [the1812/Bilibili-Evolved#3295] [提问 / Q&A] [answered]
安装的组件太多了，  有的用不到， 有的失效，综合起来貌似导致了一些加载迟缓， 但是没有批量删除/卸载的样子。
想要清除该插件的所有组件及配置文件

**Comment by the1812** [comment, marked as answer]
编辑脚本, 把存储改成只剩一对 `{}`, 保存后所有设置就重置了
![image](https://user-images.githubusercontent.com/26504152/167153604-5b26d671-eadf-4986-9998-41becb04a6d1.png)

**Comment by aswirl4** [comment]
the1812 开发大大的方法需要开启油猴的advanced设置mode
也可以在tampermonkey里删除脚本并继续去回收站（那个没有名字的小垃圾桶图标选项）里再次删除。
